portal-web/test/functional/com/liferay/portalweb/tests/enduser/staging/usecase/StagingUsecaseWithVersioning.testcase

```diff
@@ -806,7 +806,7 @@ definition {
 		Staging.viewNoPublishContentCountViaPortletStagingCP(stagingDateRange = "From Last Publish Date");
 	}
 
-	@priority = 4
+	@priority = 3
```

Comment:
This is a kind of smoke test for publication, it's more reasonable to be FP4.

Reply:
Set to FP4